### DISCUSSION TITLE
Value not displaying on multiple Combobox

### DISCUSSION THREAD

**Discussion by Emokores256** [tailwindlabs/headlessui#1510] [Help]
Is there a way to make the value of the Combobox show when using `multiple`? It shows in the dropdown that I have selected values but the values do not show in the input. I don't know whether I'm doing this wrong or if it is just a bug.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

You have to use the `displayValue` prop on the `Combobox.Input` component. The `displayValue` receives the current active option(s) and you have to return a string from those options. Example: https://headlessui.dev/react/combobox#basic-example

**Reply by roydigerhund** [reply]
Based on the example for multiple support https://headlessui.dev/react/combobox#selecting-multiple-values displayValue will always return only one option and not options.

**Reply by roydigerhund** [reply]
I tested the behavior in this example https://codesandbox.io/s/combobox-multiple-l5knvj and it seems like `displayValue` is not receiving any option values in a multiple Combobox.

But it can be helpful for displaying the query instead. Because when clicking on an option in the dropdown the input field will be emptied but the onChange is not triggered. By setting `displayValue={(person) => query}` the state still matches.
